frontend/src/components/ProductCard.jsx

```diff
@@ -96,6 +96,16 @@ const ProductCard = ({ id, image, name, price, onEditClick, onEditSubmit, onDele
                                                 </Button>
                                                     <Button
                                                         onClick={async () => {
+                                                          const isEmpty =
+                                                            updatedProduct.name.trim() === "" ||
```

Comment:
Yes this is also very good. A lot of people tend to miss trimming when doing business logic checks. Good call!